nupic/research/frameworks/vernon/run.py

```diff
@@ -133,11 +133,12 @@ def head(_, config, world_size, logger, on_checkpoint, queue, results):
             epoch_result = worker_results[epoch]
             if len(epoch_result) == world_size:
                 epoch_result = experiment_class.aggregate_results(epoch_result)
-                epoch_result.update(epoch=epoch,
-                                    timestamp=time.time(),
-                                    pid=os.getpid(),
-                                    hostname=os.uname()[1],
```

Comment:
The multi-lining style in this file is consistent with the rest of nupic.research, so you don't need whatever style tool you used (possible black). You can revert all multi-lining changes in this file to the way they originally were.

Reply:
Done